roles/model/templates/model.deployment.yaml.j2

```diff
@@ -233,6 +244,11 @@ spec:
             mountPath: /etc/pki/ca-trust/source/anchors/bundle-ca.crt
             subPath: bundle-ca.crt
             readOnly: true
+{% endif %}
+{% if is_openshift %}
+        volumeMounts:
```

Comment:
If both `bundle_ca_cert` and `is_openshift` are `true` this will render invalid YAML:
```
...
{% if bundle_ca_crt %}
        volumeMounts:
          - name: "ca-trust-extracted"
            mountPath: "/etc/pki/ca-trust/extracted"
          - name: "{{ ansible_operator_meta.name }}-bundle-cacert"
            mountPath: /etc/pki/ca-trust/source/anchors/bundle-ca.crt
            subPath: bundle-ca.crt
            readOnly: true
{% endif %}
{% if is_openshift %}
        volumeMounts:
          - name: server-tls-certs
            mountPath: /var/run/secrets/kubernetes.io/serviceaccount/
{% endif %}
...
```
`volumeMounts` is duplicated and will likely cause an error.